lib/install/validate/validator.go

```diff
@@ -470,6 +471,22 @@ func (v *Validator) credentials(op trace.Operation, input *data.Data, conf *conf
 		}
 	}
 
+	// If Grant Perms is set trying adding ReadOnly role to the Datacenter
+	// for the ops-user. This is necessary since the Login operation below
+	// fails if the ops-user has no permissions.
+	//
+	// FIXME ATC DEBT.
```

Comment:
Is this copied from the other place I put this? If so, can you add the issue link here and add a link to this line in the comments of the issue? And remove my initials from this one haha

Reply:
LOL, I thought it was an acronym for a particular type of TECH DEBT